log-manager/src/main/java/io/airlift/log/JsonFormatter.java

```diff
@@ -0,0 +1,35 @@
+package io.airlift.log;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.airlift.json.ObjectMapperProvider;
+
+import java.time.Instant;
+import java.util.logging.Formatter;
+import java.util.logging.LogRecord;
+
+public class JsonFormatter
+        extends Formatter
+{
+    private static ObjectMapper mapper = new ObjectMapperProvider().get();
```

Comment:
This can be
```java
private static final JsonCodec<JsonRecord> CODEC = new JsonCodecFactory(new ObjectMapperProvider()).jsonCodec(JsonRecord.class);